php/admin/migration/class-submenu.php

```diff
@@ -36,23 +36,14 @@ class Submenu extends Component_Abstract {
 	 *
 	 * @var string
 	 */
-	const QUERY_VAR_DEACTIVATE_AND_ACTIVATE = 'bl_deactivate_and_activate';
+	const QUERY_VAR_DEACTIVATE_AND_GCB_PAGE = 'bl_deactivate_and_gcb';
 
 	/**
 	 * The query var to deactivate this plugin and activate the new one.
 	 *
 	 * @var string
 	 */
-	const NONCE_ACTION_DEACTIVATE_AND_ACTIVATE = 'deactivate_bl_and_activate_new';
-
-	/**
-	 * The query var to disable onboarding in Genesis Custom Blocks.
-	 *
-	 * The user will have already created blocks, so onboarding doesn't seem useful.
-	 *
-	 * @var string
-	 */
-	const QUERY_VAR_DISABLE_ONBOARDING = 'disable_onboarding';
```

Comment:
This shouldn't be needed, as this activates GCB 'silently' via `activate_plugin()`, so it won't run the activation hook.